docs/source/installation/mac.rst

```diff
@@ -15,6 +15,16 @@ To install cairo:
 
    brew install cairo
 
+To install Pango and it dependencies:
```

Comment:
Thank you SO much for adding all these instructinos.

Reply:
It should definitely should be part o this PR. ;-)